.ci/docker/requirements-docs.txt

```diff
@@ -49,8 +49,8 @@ IPython==8.12.0
 #Pinned versions: 8.12.0
 
 myst-nb==0.17.2
-#Description: This is used to generate PyTorch functorch docs
-#Pinned versions: 0.13.2
+#Description: This is used to generate PyTorch functorch and torch.compile docs
```

Comment:
We probably want to skip this change since it will trigger docker updates. Something we don't want to do at this point